packages/frontend-shared/src/graphql/urqlClient.ts

```diff
@@ -31,6 +31,7 @@ declare global {
     __CYPRESS_INITIAL_DATA__: string
     __CYPRESS_MODE__: 'run' | 'open'
     __RUN_MODE_SPECS__: SpecFile[]
+    __CYPRESS_TESTING_TYPE__: 'e2e' | 'component'
```

Comment:
This might be a sign we should move the declaration of this type out of this file, it's added here so that TS won't complain that `Property '__CYPRESS_TESTING_TYPE__' does not exist on type 'Window & typeof globalThis'`